doc/spec_TransferElement.dox

```diff
@@ -293,6 +294,8 @@ This documnent is currently still **INCOMPLETE**!
 
 
 ### (*) Comments ###
+- \anchor transferElement_comment_B_2_5_1  \ref transferElement_B_2_5_1 "2.5.1" Backends which don't support data validity flags (like the PCIe backend) just get the clampled value. The DOOCS backend however, which supports data validity, sets the outgoing data to invalid in addition.
```

Comment:
I am actually not sure whether DOOCS supports flagging data as invalid in the writing direction... Maybe I would not name specific protocols here but keep the statement general.